extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -39,6 +39,8 @@ export class AmpExperiment extends AMP.BaseElement {
     if (!this.isExperimentOn_) {
       dev().warn(EXPERIMENT, `Experiment ${EXPERIMENT} disabled`);
       toggle(this.element, false);
+      getService(this.win, 'variant',
+          () => Promise.reject(EXPERIMENT + ' disabled'));
```

Comment:
Should we resolve this instead? This will cause the body to immediately (next-tick) become visible instead of waiting for the other render-delaying extensions to also load.